backend/main.py

```
@@ -482,6 +533,31 @@ async def api_clone_repo(request: Request):
+                        github_repo_id=body.get("github_repo_id", 0),
+                        name=repo_name,
+                        full_name=body.get("full_name", f"{db_user['login']}/{repo_name}"),
+                        clone_url=clone_url,
+                        html_url=body.get("html_url"),
+                        description=body.get("description"),
+                        private=body.get("private", False),
+                        language=body.get("language"),
+                        default_branch=body.get("default_branch", "main"),
+                    )
+                    logger.info(f"Repository registered in database: {repo_name}")
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)